### PR TITLE
Refactor the Run a Node

### LINKED ISSUE #245: Docs update request: /node-running/gentle-introduction-run-node
Source: https://developer.arbitrum.io/node-running/gentle-introduction-run-node

Request: If that page of the doc isn't written you could eventually link to an introduction, it was most likely written somewhere already in a blog post or in a twitter thread

### PR DESCRIPTION
**Pull request by mahsamoosavi**
1. Add a new page; introduction to Arbitrum nodes that will replace [this](https://developer.arbitrum.io/node-running/gentle-introduction-run-node)
2. Remove the Concepts subpage (contents will be defined succinctly with a glossary entry)
3. Cleanup titles in all the pages to follow our [styling guides](https://www.notion.so/arbitrum/Style-convention-terminology-diagramming-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c)
4. Fix typos
5. close #245 

[[preview](https://nitro-docs-k8cqazo67-offchain-labs.vercel.app/node-running/running-a-node)]

### CODE REVIEW

File: arbitrum-docs/node-running/gentle-introduction-run-node.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
 ---
-title: "A gentle introduction to Arbitrum nodes"
+title: "A gentle Introduction to Arbitrum Nodes"
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/gentle-introduction-run-node.md:2]: Shouldn't `introduction` and `nodes` be in lowercase here?

File: arbitrum-docs/node-running/gentle-introduction-run-node.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 sidebar_label: "A gentle introduction: Arbitrum nodes"
 description: "Arbitrum is a suite of Ethereum L2 scaling solutions supported by a decentralized network of nodes. This guide introduces you to Arbitrum's node types and how they work together to scale Ethereum."
-author: dzgoldman
+author: mahsamoosavi
 ---
 
-import UnderConstructionPartial from '../partials/_under-construction-banner-partial.md'; 
+In order to be able to _ineract with_ or _build applications on_ any of the Arbitrum chains, you need access to the corresponding Arbitrum node. Options are:
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/gentle-introduction-run-node.md:8]:
```suggestion
In order to be able to _interact with_ or _build applications on_ any of the Arbitrum chains, you need access to the corresponding Arbitrum node. Options are:
```

File: arbitrum-docs/node-running/gentle-introduction-run-node.md [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
 
-<!-- todo: build bridge-specific gentle intro, and reuse content from the OG gentle intro whenever possible 
-
-import GentleIntroPartial from '../partials/_gentle-intro-partial.md';
-
-<GentleIntroPartial />
-
--->
\ No newline at end of file
+- Step-by-step instructions for running different Arbitrum nodes
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/gentle-introduction-run-node.md:15]: I would add links here. For instructions there are several pages, not sure how to handle it, but maybe you can list them after a colon, and place individual links for the most common type of nodes like ": full Nitro node, full Classic node, local dev node, feed relay, and others."
Also, with the new changes there aren't concepts in this section. We can remove that line now and add it back later if we create a concept page.

File: arbitrum-docs/das/daserver-instructions.mdx [hunk 4]
```diff
@@ -1,4 +1,10 @@
-# Data Availability Server Instructions
+---
+title: "Run a Data Availability Server"
```
Comment (TucksonDev) [inline arbitrum-docs/das/daserver-instructions.mdx:2]:
```suggestion
title: "How to run a Data Availability Server"
```

@symbolpunk can confirm here, but I think the convention was to use "How to ...." in the title of the page, while the sidebar would just mention the action.

File: arbitrum-docs/node-running/build-nitro-locally.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "How to build Nitro locally (Docker, Debian)"
+title: "Build nitro locally (Docker, Debian)"
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/build-nitro-locally.md:2]: Same here, I would leave the "How to build Nitro" (also Nitro in uppercase)

File: arbitrum-docs/node-running/how-tos/read-sequencer-feed.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "How to read the sequencer feed"
+title: "How to Read the Sequencer Feed?"
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/how-tos/read-sequencer-feed.md:2]: I would leave it as "How to read the sequencer feed" (without question mark). Not sure about Sequencer being uppercase or lowercase though.

File: arbitrum-docs/node-running/running-a-node.md [hunk 7]
```diff
@@ -7,7 +7,7 @@ reader-task: run a node with minimal effort and maximum understanding
 content-type: quickstart
 ---
 
-🔉 Note: If you’re interested in accessing an Arbitrum chain, but you don’t want to set up your own node, see our [Node Providers](./node-providers.mdx) to get RPC access to fully-managed nodes hosted by a third party provider.
+⚠️ Note: There is no protocol level incentive to run an Arbitum full node. If you’re interested in accessing an Arbitrum chain, but you don’t want to set up your own node, see our [Node Providers](./node-providers.mdx) to get RPC access to fully-managed nodes hosted by a third party provider.
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/running-a-node.md:10]: Good idea. I like this icon better than the speaker :)

File: arbitrum-docs/node-running/running-a-validator.mdx [hunk 8]
```diff
@@ -20,7 +24,7 @@ Here we describe different types of validators and provide instructions on how t
     - Continuously create new assertions, challenging any bad assertions found (wallet required, always staked, most expensive node to run)
     - Note that if there is more than one `MakeNodes` validator running, they might all try to create a new assertion at same time. In that case, only one will be successful, and the others will have still spent gas on reverted calls that didn't do anything.
 
-### Running Watchtower Validator
+### Running watchtower validator
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/running-a-validator.mdx:27]: Not sure about "Watchtower". I lean towards having it in uppercase, but idk 🤔

File: arbitrum-docs/node-running/running-an-archive-node.mdx [hunk 9]
```diff
@@ -1,13 +1,13 @@
 ---
-title: 'How to run an archive node'
+title: 'How to run an Archive node'
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/running-an-archive-node.mdx:2]: I think we use "archive" in lowercase throughout the rest of the pages. Should we leave it in lowercase?

File: arbitrum-docs/node-running/running-an-archive-node.mdx [hunk 10]
```diff
@@ -66,7 +66,7 @@ Note that other chains don't have Classic blocks, and thus don't require an init
 | `--node.cache.allow-slow-lookup`            | Required for running an **Arbitrum Classic** archival node. When this option is present, will load old blocks from disk if not in memory cache. |
 | `--core.checkpoint-gas-frequency=156250000` | Required for running an **Arbitrum Classic** archival node.                                                                                     |
 
-### Run the Docker image(s)
+### Run the docker image(s)
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/running-an-archive-node.mdx:69]: Docker is also usually in uppercase, should we leave it like that?

File: arbitrum-docs/node-running/gentle-introduction-run-node.md [hunk 11]
```diff
@@ -2,18 +2,17 @@
...
-
-import GentleIntroPartial from '../partials/_gentle-intro-partial.md';
-
-<GentleIntroPartial />
-
--->
\ No newline at end of file
+- Step-by-step instructions for running different Arbitrum nodes, including [full Nitro node](./running-a-node.md), [full Classic node](./how-tos/running-a-classic-node.mdx), [local dev node](./how-tos/local-dev-node.mdx), [feed relay](./how-tos/running-a-feed-relay.mdx), and [validator](./how-tos/running-a-validator.mdx)
+- Step-by-step instructions for how to [read the sequencer feed](./how-tos/read-sequencer-feed.md), [build the Nitro locally](./how-tos/build-nitro-locally.md), and [run a DAS](../das/daserver-instructions.mdx)
+- Troubleshooting page
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/gentle-introduction-run-node.md:17]: Should we add links to these ones too? (Troubleshooting and FAQ)

File: arbitrum-docs/node-running/how-tos/running-a-feed-relay.mdx [hunk 12]
```diff
@@ -12,4 +16,4 @@ When running more than one node, you want to run a single feed relay per datacen
   ```
 
 
-Note that Arbitrum classic does not communicate with Nitro sequencer, so classic relay is no longer used.
\ No newline at end of file
+⚠️ Note that Arbitrum classic does not communicate with Nitro sequencer, so classic relay is no longer used.
```
Comment (TucksonDev) [inline arbitrum-docs/node-running/how-tos/running-a-feed-relay.mdx:19]: Minor detail: this doesn't sound like a warning, but more like a notice. Should we use the speaker here, or the "i" for information icon, or no icon?

File: arbitrum-docs/partials/_glossary-partial.md [hunk 13]
```diff
@@ -57,7 +57,7 @@
 ### Blockchain {#blockchain}
 <p>A distributed digital ledger that is used to record transactions and store data in a secure, transparent, and tamper-resistant way, notably in cryptocurrency protocols. </p>
 
-### 🧢 CAP Finance {#cap-finance}
```
Comment (TucksonDev) [inline arbitrum-docs/partials/_glossary-partial.md:60]: It looks like your notion-docs package is not up to date (it shouldn't remove these icons). Can you run `yarn install` inside that folder and re-run `upgrade:all`?

**Comment by TucksonDev** [comment]
There are still a few broken links. You can verify those running docusaurus in your local (`yarn start` inside the website folder). Here's the warnings I get:
```
[WARNING] Docs markdown link couldn't be resolved: (/node-running/local-dev-node.mdx) in "/arbitrum-docs/arbitrum-docs/faqs/nodes-faqs.mdx" for version current
[WARNING] Docs markdown link couldn't be resolved: (./node-providers.mdx) in "/arbitrum-docs/arbitrum-docs/node-running/how-tos/running-a-classic-node.mdx" for version current
[WARNING] Docs markdown link couldn't be resolved: (../migration/state-migration.mdx) in "/arbitrum-docs/arbitrum-docs/node-running/how-tos/running-an-archive-node.mdx" for version current
[WARNING] Docs markdown link couldn't be resolved: (./node-providers.mdx) in "/arbitrum-docs/arbitrum-docs/node-running/how-tos/running-an-archive-node.mdx" for version current
[WARNING] Docs markdown link couldn't be resolved: (./running-a-node.md) in "/arbitrum-docs/arbitrum-docs/node-running/how-tos/running-an-archive-node.mdx" for version current
[WARNING] Docs markdown link couldn't be resolved: (./running-a-node.md) in "/arbitrum-docs/arbitrum-docs/node-running/how-tos/running-an-archive-node.mdx" for version current
[WARNING] Docs markdown link couldn't be resolved: (./node-providers.mdx) in "/arbitrum-docs/arbitrum-docs/node-running/how-tos/running-an-archive-node.mdx" for version current
[WARNING] Docs markdown link couldn't be resolved: (../public-chains.mdx) in "/arbitrum-docs/arbitrum-docs/node-running/how-tos/running-an-archive-node.mdx" for version current
```

**Comment by TucksonDev** [comment]
Finally, to avoid external references get a 404 when accessing a URL we have moved, we should add the URL changes to the `oldPathToNewPath` object in `website/src/pages/index.js`.

You can add them at the end, like so:
`"/node-running/running-a-classic-node": "/node-running/how-tos/running-a-classic-node"`

**Review by TucksonDev — CHANGES_REQUESTED** [review]
The only important things are the broken links and the missing icons in the glossary!

File: arbitrum-docs/node-running/how-tos/running-a-feed-relay.mdx (hunk 12 above)
Comment (mahsamoosavi) [reply arbitrum-docs/node-running/how-tos/running-a-feed-relay.mdx:19]: I'm ok with having no icon since it's just an info. Sounds good?

**Review by TucksonDev — APPROVED** [review]
LGTM! 🚀

**Pull request merged**